src/Search.elm

```diff
@@ -512,7 +512,13 @@ view { toRoute, categoryName } title model viewSuccess outMsg =
 
                 RemoteData.Success result ->
                     if result.hits.total.value == 0 then
-                        [ h4 [] [ text <| "No " ++ categoryName ++ " found!" ] ]
+                        [ h4 [] [ text <| "No " ++ categoryName ++ " found!" ]
+                        , text "How to "
+                        , Html.a [ href "https://nixos.org/manual/nixpkgs/stable/#chap-quick-start"] [ text "add" ]
+                        , text " or "
```

Comment:
I changed the copy from `/` to ` or ` as it it makes more clear that there are 2 separate actions.